assets/msalv2.js

```diff
@@ -175,11 +175,16 @@ var aadOauth = (function () {
     return authResult ? authResult.idToken : null;
   }
 
+  function isLogged() {
+    return getAccount() !== undefined;
```

Comment:
Hi @Zazo032,

I've debugged the web version and when the plugin calls getAccount without being authenticated, it returns the value `null`. Changing this line to something similar to `return getAccount() != null;` should solve the problem (I'm not a web expert but I think this will check for both, undefined and null).

Can you reproduce this issue in your app?